{{cookiecutter.app_name}}/webpack.config.js

```diff
@@ -0,0 +1,63 @@
+const webpack = require('webpack');
+
+/*
+ * Webpack Plugins
+ */
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const ManifestRevisionPlugin = require('manifest-revision-webpack-plugin');
+
+// take debug mode from the environment
+const debug = (process.env.NODE_ENV !== 'prod');
+
+// Development asset host (webpack dev server)
+const publicHost = debug ? 'http://localhost:2992' : '';
+
+const rootAssetPath = './assets/';
+
+module.exports = {
+  // configuration
+  context: __dirname,
+  entry: {
+    main_js: './assets/js/main',
+    main_css: [
+      './node_modules/font-awesome/css/font-awesome.css',
+      './node_modules/bootstrap/dist/css/bootstrap.css',
+      './assets/css/style.css'
+    ]
+  },
+  output: {
+    path: __dirname + '/{{cookiecutter.app_name}}/static/build',
+    publicPath: publicHost + '/static/build/',
+    filename: '[name].[hash].js',
+    chunkFilename: '[id].[hash].js'
+  },
+  resolve: {
+    extensions: ['.js', '.jsx', '.css']
+  },
+  devtool: debug ? '#inline-sourcemap' : false,
```

Comment:
How about using `source-map` in production mode? [It doesn't hurt](https://twitter.com/dan_abramov/status/866039354740834304), and can be very useful for debugging.